source/NVDAObjects/window/excel.py

```diff
@@ -1155,6 +1156,15 @@ def _get_states(self):
 			states.add(controlTypes.STATE_UNLOCKED)
 		return states
 
+	def event_typedCharacter(self,ch):
+		# # 6570: You cannot type into protected cells.
+		# A part from speaking characters being miss-leading, Office 2016 protected view doubles characters as well.
+		# Therefore for any non-control char on protected cells, play the default sound rather than speaking the character
+		if ord(ch)>26 and controlTypes.STATE_UNLOCKED not in self.states and controlTypes.STATE_PROTECTED in self.parent.states: 

```

Comment:
It would be helpful if 26 was a named value. Is it the first printable character code?